cloudify-manager-worker/templates/statefulset.yaml

```diff
@@ -255,7 +255,45 @@ spec:
         {{- end }}
         {{- end }}
         {{- end }}
+      {{- if .Values.fastly.enabled }}
+      - name: sigsci-agent
+        image: {{ include "fastly.image" . }}
+        imagePullPolicy: Always
+        env:
+        - name: SIGSCI_ACCESSKEYID
+          valueFrom:
+            secretKeyRef:
+              name: fastly-sigsci
```

Comment:
Should you create this secret by helm too or you want to use preliminary created?

Reply:
I want to use precreated